src/app.py

```diff
@@ -17,13 +18,32 @@
 def hello():
     return "Hello World!"
 
-@app.route("/sessions")      
-def home_page():
-    sessions = mongo.db.sessions
+@app.route("/sessions",methods=["POST"])      
+def sessions():
+    data = request.get_json()
+    session_collection = mongo.db.sessions
+    
+    new_session = Session( data["user_id"],data["first_received_at"],data["focus_score"],data["transcribed_at"],data["transcribed_speech"])
+    print(new_session.__dict__)
+    session_collection.insert(new_session.__dict__)
+    # session_collection.insert({
+    #     "user_id":data["user_id"],
+    #     "first_received_at": data["first_received_at"],
+    #     "focus_score": data["focus_score"],
+    #     "transcribed_at": data["transcribed_at"],
+    #     "transcribed_speech": data["transcribed_speech"]
+    # })
     output = []
-    for session in sessions.find():
+    for session in session_collection.find():
         print(session)
```

Comment:
printing here is okay, but make sure to make it traceable by adding a statement of what we're printing. Ex. `print("NEW SESSION OBJECT CREATED: ", session)`